platform/platform-manager.js

```diff
@@ -0,0 +1,97 @@
+const { fork } = require('child_process');
+const fs = require("fs-extra");
+let ipc = require('node-ipc');
+
+let services = {
+  "app-manager": {
+    path: __dirname + "/app-manager/app-manager.js",
+    socket: undefined,
+    process: undefined
+  },
+  "api-server": {
+    path: __dirname + "/api-server/server.js",
+    socket: undefined,
+    process: undefined
+  }
+};
+
+// data = {
```

Comment:
Write a statement "data is of the format as described". To make it more readable.